FlowCryptAppTests/Functionality/Services/ComposeMessageServiceTests.swift

```diff
@@ -207,7 +209,7 @@ class ComposeMessageServiceTests: XCTestCase {
 
     func testValidateMessageInputWithAllEmptyRecipientPubKeys() async {
         encryptedStorage.getKeypairsResult = [keypair]
-        for recipient in recipients {
+        for _ in recipients {
             localContactsProvider.retrievePubKeysResult = { _ in
```

Comment:
I suppose the `for _ in recipients` can be removed here and a few more times below. Don't see reason to call `ocalContactsProvider.retrievePubKeysResult =` more than once.

Reply:
Yeah, good point - no need to call it for each recipient 👍